src/test/regress/expected/multi_explain.out

```diff
@@ -497,19 +497,16 @@ EXPLAIN (COSTS FALSE)
 	WHERE l_orderkey = o_orderkey
 	AND o_custkey = c_custkey
 	AND l_suppkey = s_suppkey;
-Distributed Query into pg_merge_job_570021
+Distributed Query into pg_merge_job_570020
   Executor: Task-Tracker
   Task Count: 1
   Tasks Shown: None, not supported for re-partition queries
   ->  MapMergeJob
-        Map Task Count: 1
+        Map Task Count: 8
         Merge Task Count: 1
-        ->  MapMergeJob
-              Map Task Count: 8
-              Merge Task Count: 1
```

Comment:
To be more specific, because we've replaced `supplier` to a reference table, it is always planned to be a broadcast join. 
What about  I create another table for this specific test say `supplier_single_shard_hash` and use it? Does that work for you?